RadixWallet/Features/SettingsFeature/DisplayMnemonics/Coordinator/DisplayMnemonics.swift

```diff
@@ -79,11 +79,20 @@ public struct DisplayMnemonics: Sendable, FeatureReducer {
 						// `nil` means read profile in ProfileStore, instead of using an overriding profile
 						nil
 					)
-					let deviceFactorSources = entitiesForDeviceFactorSources.map {
-						DisplayEntitiesControlledByMnemonic.State(
-							accountsForDeviceFactorSource: $0,
-							mode: $0.isMnemonicPresentInKeychain ? .mnemonicCanBeDisplayed : .mnemonicNeedsImport
-						)
+					let deviceFactorSources: [DisplayEntitiesControlledByMnemonic.State] = entitiesForDeviceFactorSources.flatMap { ents in
```

Comment:
I know, "clarity over brevity" Im a strong proponent of that, why you get to enjoy my 40 chars long identifiers! but I chose `ents` here over `entitiesControlledByFactorSource` since I used it 6 times and it would clutter this `flatMap` also I cannot use `$0` only because we have nested `map` inside....

Reply:
`entities` is also pretty short, and actually describes what it is...

Reply:
no, `entities` make it sound like the thing contains only entities, this contains the FactorSource and the entities.